chain/jsonrpc/res/chunk_info.html

```diff
@@ -0,0 +1,91 @@
+<html>
+<head>
+    <style>
+        table {
+            width: 100%;
+            border-collapse: collapse;
+        }
+
+        table,
+        th,
+        td {
+            border: 1px solid black;
+        }
+
+        td {
+            text-align: left;
+            vertical-align: top;
+            padding: 8px;
+        }
+
+        th {
+            text-align: center;
+            vertical-align: center;
+            padding: 8px;
+            background-color: lightgrey;
+        }
+
+        tr.active {
+            background-color: #eff8bf;
+        }
+    </style>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script>
+        $(document).ready(() => {
+            $('span').text("Loading...");
+            $.ajax({
+                type: "GET",
+                url: "/debug/api/status",
+                success: data => {
+                    let chunk_info = data.detailed_debug_status.chunk_info;
+                    $('.js-block-count').text(chunk_info.num_of_blocks_in_progress);
+                    $('.js-chunk-count').text(chunk_info.num_of_chunks_in_progress);
+                    $('.js-orphan-count').text(chunk_info.num_of_orphans);
+                    chunk_info.next_blocks_log.forEach((log, index) =>
+                        $('.js-tbody-validators').append($('<tr>')
```

Comment:
tbody-validators? or should it be 'tbody-log'?

Reply:
Good catch! It should be `.js-tbody-log`.